gobot/cmd/root.go

```diff
@@ -117,19 +123,28 @@ func run(logger *zap.SugaredLogger) error {
 		Maintainers:    Maintainers,
 	}
 
-	prHandler := &handlers.PullRequestHandler{
+	prHandler := &handlers.PullRequestEventHandler{
 		ClientCreator:  cc,
 		Logger:         logger,
 		RequiredLabels: RequiredLabels,
 		BotUsername:    BotUsername,
 		Maintainers:    Maintainers,
 	}
 
+	prCreateHandler := &handlers.PullRequestCreateHandler{
+		ClientCreator:  cc,
+		Logger:         logger,
+		TaxonomyRepo:   TaxonomyRepo,
+		GithubUsername: GithubUsername,
+		GithubToken:    GithubToken,
+	}
+
 	webhookHandler := githubapp.NewDefaultEventDispatcher(ghConfig, prCommentHandler, prHandler)
 
 	http.Handle(githubapp.DefaultWebhookRoute, webhookHandler)
 
-	addr := net.JoinHostPort(HTTPAddress, strconv.Itoa(HTTPPort))
+	//addr := net.JoinHostPort(HTTPAddress, strconv.Itoa(HTTPPort))
```

Comment:
```suggestion
```

Reply:
I will fix it through new PR, because the original changes were merged through another PR.